.travis.yml

```diff
@@ -1,26 +1,42 @@
-sudo: false
+sudo: required
 language: python
 python: 2.7
-addons:
-  firefox: "latest"
-node: '0.10'
 env:
-  globals:
-  - DJANGO_LIVE_TEST_SERVER_ADDRESS="localhost:9900-9999"
-  - PIP_DOWNLOAD_CACHE="pip_cache"
-  matrix:
-  - TEST_SUITE=lint
-#  - TEST_SUITE=ui MARK_EXPRESSION=smoke
-  - TEST_SUITE=django
-  - TEST_SUITE=docker
-matrix:
-  fast_finish: true
-cache:
-  directories:
-  - /home/travis/virtualenv
-install: travis_retry scripts/travis/install.sh
-before_script: travis_retry scripts/travis/setup.sh
-script: scripts/travis/dispatch.sh $TEST_SUITE
+  global:
+    - DEBUG=False
+    - STAGE=False
+    - SESSION_COOKIE_SECURE=False
+    - CELERY_ALWAYS_EAGER=True
+    - AXES_BEHIND_REVERSE_PROXY=False
+    - PIPELINE_ENABLED=True
+    - ES_LIVE_INDEXING=False
+    - ALLOWED_HOSTS=*
+    - BROKER_URL=redis://redis:643
+    - REDIS_DEFAULT_URL=redis://redis:6379/1
+    - REDIS_HELPFULVOTES_URL=redis://redis:6379/2
+    - CACHE_URL=redis://redis:6379/3
+    - CSRF_COOKIE_SECURE=False
+    - DATABASE_URL=mysql://root:kitsune@mariadb:3306/kitsune
+    - DOMAIN=localhost
+    - ES_URLS=elasticsearch:9200
+    - SITE_URL=http://localhost:8000
+    - SECRET_KEY=secret
+services:
+  - docker
+install:
+  - pip install -U docker-compose
+  - env > .env
+  - docker-compose up -d mariadb
+  - docker-compose up -d elasticsearch
+  - docker-compose up -d redis
+  - docker-compose build
+  - docker-compose run --user root web ./manage.py nunjucks_precompile
```

Comment:
rathar than running as root you can export `UID` into environment variable and it will run with the host uid. So there will be no problem in creating files.
like
```
export UID
docker-compose run web ./manage.py nunjucks_precompile
```

Reply:
tried that without luck in https://travis-ci.org/mozilla/kitsune/builds/305240903

I 'll welcome a PR to fix this.